docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -462,17 +462,28 @@ This would result in::
     b: patch
     c: default
 
-If ``list_merge='replace'`` (the default), arrays from the right hash will "replace" the ones in the left hash::
+If ``list_merge='combine'`` (SHOULD be the default), arrays from the right hash will "combine" with the ones in the left hash in exactly the same way keys in hashes "combine" (i.e. this option essentially treats arrays as hashes with numeric keys)::
```

Comment:
'SHOULD be' the default seems rather subjective since the lists need to be in a predictable order.

```suggestion
If ``list_merge='combine'`` (not the default), arrays from the right hash will "combine" with the ones in the left hash in exactly the same way keys in hashes "combine" (i.e. this option essentially treats arrays as hashes with numeric keys)::
```

Reply:
I agree.  I just couldn't resist... 😈